linux/scripts/upload-to-debian.sh

```diff
@@ -101,11 +101,21 @@ if [ -n "$PUSH" ]; then
     $NOOP git push origin chore/linux/changelog
 fi
 
-git checkout -B chore/linux/cherry-pick/changelog origin/master
+if git branch -r | grep origin/beta; then
```

Comment:
Since l. 34 sources `build-utils.sh`, does this script have access to `$TIER`?

Reply:
Yep, just wondering what branch this script runs on? Is it running on a variety of branches? If so, then yep, we might need to check for existence of origin/beta branch as @ermshiperete has done here.

Reply:
Yes, the intention is that it doesn't matter what branch you're on, it will always upload the latest stable release to Debian.